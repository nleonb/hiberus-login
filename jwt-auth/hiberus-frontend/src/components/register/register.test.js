import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from "react-router-dom";

import Register from './register';

describe('Test Register component ', () => {
    it('renders correctly', () => {
        const component = renderer.create(
            <MemoryRouter initialentries="{['/']}">
                <Register />
            </MemoryRouter>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});