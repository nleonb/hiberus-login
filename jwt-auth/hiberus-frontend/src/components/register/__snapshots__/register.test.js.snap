// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Register component  renders correctly 1`] = `
<section
  className="hero has-background-grey-light is-fullheight is-fullwidth"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <div
        className="columns is-centered"
      >
        <div
          className="column is-4-desktop"
        >
          <form
            className="box"
            onSubmit={[Function]}
          >
            
            <div
              className="field mt-5"
            >
              <label
                className="label"
              >
                Name
              </label>
              <div
                className="controls"
              >
                <input
                  className="input"
                  name="name"
                  onChange={[Function]}
                  placeholder="Name"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              className="field mt-5"
            >
              <label
                className="label"
              >
                Surname
              </label>
              <div
                className="controls"
              >
                <input
                  className="input"
                  name="surname"
                  onChange={[Function]}
                  placeholder="Surname"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              className="field mt-5"
            >
              <label
                className="label"
              >
                Email
              </label>
              <div
                className="controls"
              >
                <input
                  className="input"
                  name="email"
                  onChange={[Function]}
                  placeholder="Email"
                  type="email"
                  value=""
                />
              </div>
            </div>
            <div
              className="field mt-5"
            >
              <label
                className="label"
              >
                Password
              </label>
              <div
                className="controls"
              >
                <input
                  className="input"
                  name="password"
                  onChange={[Function]}
                  placeholder="******"
                  type="password"
                  value=""
                />
              </div>
            </div>
            <div
              className="field mt-5"
            >
              <label
                className="label"
              >
                Confirm Password
              </label>
              <div
                className="controls"
              >
                <input
                  className="input"
                  name="confPassword"
                  onChange={[Function]}
                  placeholder="******"
                  type="password"
                  value=""
                />
              </div>
            </div>
            <div
              className="field mt-5"
            >
              <button
                className="button is-success is-fullwidth"
              >
                Register
              </button>
            </div>
            <div
              className="field mt-5"
            >
              <button
                className="button is-link is-fullwidth"
                onClick={[Function]}
              >
                Back to Login
              </button>
            </div>
          </form>
        </div>
      </div>
    </div>
  </div>
</section>
`;
